pkg/dashboard/static/index.html

```diff
@@ -135,7 +135,7 @@ <h4 class="mt-4">Namespaces</h4>
 
         <!-- INSTALLED LIST -->
         <div class="col ms-2" id="installedList">
-            <div class="col rounded rounded-1 b-shadow header">
+            <div class="col rounded rounded-1 header">
```

Comment:
The default shadow of that block is OK. The problem is with `:hover` shadow that is set in our `styles.css`. Adding `.body` class to `#installedList .body .b-shadow:hover` will fix the issue.

Reply:
done